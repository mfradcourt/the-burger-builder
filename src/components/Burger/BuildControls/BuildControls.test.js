import React from 'react';
import { shallow } from "enzyme";

import BuildControls from './BuildControls';

describe('<BuildControls />', () => {
  let wrapper;

  const disabledInfo = {
    salad: true,
    meat: true,
    cheese: false,
    bacon: false
  };

  it('should render BuildControls', () => {
    wrapper = shallow(
      <BuildControls
        disabled={disabledInfo}
        price={50}
      />
    );

    expect(wrapper).toMatchSnapshot();
 });
});
