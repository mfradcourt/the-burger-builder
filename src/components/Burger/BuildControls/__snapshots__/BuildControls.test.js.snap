// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BuildControls /> should render BuildControls 1`] = `
<div
  className="BuildControls"
>
  <p>
    Current Price: 
    <strong>
      50.00
    </strong>
  </p>
  <buildControl
    added={[Function]}
    disabled={true}
    key="Salad"
    label="Salad"
    removed={[Function]}
  />
  <buildControl
    added={[Function]}
    disabled={false}
    key="Bacon"
    label="Bacon"
    removed={[Function]}
  />
  <buildControl
    added={[Function]}
    disabled={false}
    key="Cheese"
    label="Cheese"
    removed={[Function]}
  />
  <buildControl
    added={[Function]}
    disabled={true}
    key="Meat"
    label="Meat"
    removed={[Function]}
  />
  <button
    className="OrderButton"
    disabled={true}
  >
    ORDER NOW
  </button>
</div>
`;
